dvc/repo/experiments/__init__.py

```diff
@@ -311,31 +310,18 @@ def _unpack_args(self, tree=None):
 
     def _update_params(self, params: dict):
         """Update experiment params files with the specified values."""
+        from benedict import benedict
```

Comment:
Not sure if you prefer local or global imports. Since this is only needed for the beta experiments feature, I figured local was appropriate.